feat/feature-flags.json

```diff
@@ -1,5 +1,13 @@
 {
   "features": {
+    "external_sources": {
+      "status": {
+        "staging": "switchable",
+        "production": "disabled"
+      },
+      "description": "Toggle the external sources in the header's search type switcher.",
+      "defaultState": "off"
+    },
```

Comment:
It seems like there was maybe a bad main merge. These changes are already present in main :thinking: Do you mind rebasing the PR to get rid of these duplicates?

Reply:
Sorry about it, I'm going to rebase it!